### PR TITLE
Add Pushsafer.com Notification Service

### PR DESCRIPTION
**Pull request by appzer**

### CODE REVIEW

**Comment by PoltoS** [comment]
Hello! Thank you for your contribution!

Please go thru https://developer.z-wave.me. Create an account and upload your app for revision.

Once approved, it will be visible in the Online Apps section for all users.

I would suggest to move your folder to userModules, please the attached script [uploadModule.sh](https://github.com/Z-Wave-Me/home-automation/files/10950732/uploadModule.sh.txt) there (rename to .sh and make executable) and use it to upload each new version of your app.

**Comment by PoltoS** [comment]
BTW, I see your app already in our store: http://developer.z-wave.me/?uri=public#/web/apps/496

You can upgrade that app

File: modules/Pushsafer/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+// ----------------------------------------------------------------------------
+// --- Module instance initialized
+// ----------------------------------------------------------------------------
+
+NotificationPushsafer.prototype.init = function (config) {
+    NotificationPushsafer.super_.prototype.init.call(this, config);
+
+    var self = this;
+
```
Comment (PoltoS) [inline modules/Pushsafer/index.js:31]: Would be also nice if your app could provide a notification channel to allow post events thru notification filtering.

Create the channel: https://github.com/Z-Wave-Me/home-automation/blob/master/modules/NotificationChannelEmail/index.js#L78

Unregister the channel: https://github.com/Z-Wave-Me/home-automation/blob/master/modules/NotificationChannelEmail/index.js#L106

**Comment by appzer** [comment]
> BTW, I see your app already in our store: http://developer.z-wave.me/?uri=public#/web/apps/496
> 
> You can upgrade that app

i've seen the app but i cant update it, because any of my mail addresses dont work to login the developer console.
It is possible to get the mail address which was used to publish the module?

Kevin

**Comment by PoltoS** [comment]
Is syl.m@freenet.de you?

**Comment by appzer** [comment]
> Is [syl.m@freenet.de](mailto:syl.m@freenet.de) you?

no! So someone use my code and add the plugin to the store.
It is possible to get access to this store entry to make an update?

Kevin

**Comment by PoltoS** [comment]
This was great that someone shared your opensource code with credits to you ;)

We have changed ownership to you, please go on with your credentials you created recently.

**Comment by appzer** [comment]
> This was great that someone shared your opensource code with credits to you ;)
> 
> We have changed ownership to you, please go on with your credentials you created recently.

i uploaded an update

**Comment by PoltoS** [comment]
Added as http://developer.z-wave.me/?uri=public#/web/apps/496

**Comment by PoltoS** [comment]
@appzer Thank you for your contribution!

**Pull request closed without merging**